src/server/index.ts

```diff
@@ -34,6 +35,7 @@ app.get('/encrypt', async (req: Request, res: Response) => {
   try {
     debug.log('Request to encrypt')
     await validateValues()
+    await genDrone()
```

Comment:
Perhaps we should rename this endpoint to /commit instead of /encrypt
I am also wondering if we could reuse commit.ts instead of reinventing it in the server?

Reply:
that is already started in another branch...for now I am adding this here